ops/mainnet/prod/backend/main.tf

```diff
@@ -1,7 +1,7 @@
 terraform {
   backend "s3" {
     bucket = "nxtp-terraform-mainnet-prod-backend"
-    key    = "state/"
```

Comment:
This needs to be reverted immediately, otherwise the state will be migrated

Reply:
Was getting an error "key must not end with '/'" running `terraform init`. Reverting as this will cause the migrate.